components/auth/SignInForm.tsx

```diff
@@ -93,8 +93,8 @@ export default function SignInForm() {
                     value={serverAddress}
                     type="text"
                     onChange={(e) => setServerAddress(e.target.value)}
-                    readOnly={showNextAuth}
-                  />
+                    disabled={showNextAuth}
+                    />
```

Comment:
leave here `readonly` state, just add styles to input when its read-only (i.e. `showNextAuth` is `true`).

let the styles be `bg-slate-100` for background color and `bg-slate-200` for border color.